### ISSUE TITLE
build issue: wikipedia_my_all_maxi_2021-08.zim (missing main page)

### ISSUE THREAD

**Issue by lidel** [ipfs/distributed-wikipedia-mirror#101]
```
./mirrorzim.sh --languagecode=my --wikitype=wikipedia --hostingdnsdomain=my.wikipedia-on-ipfs.org
```

Builds, but missing main page contents:

> ![2021-08-18--16-31-49](https://user-images.githubusercontent.com/157609/129917180-35a92d8c-247f-400d-aee8-15fa5020b6db.png)

**Comment by kelson42** [comment]
Indeed... http://library.kiwix.org/wikipedia_my_all_maxi/A/%E1%80%9D%E1%80%AE%E1%80%80%E1%80%AE%E1%80%95%E1%80%AE%E1%80%B8%E1%80%92%E1%80%AE%E1%80%B8%E1%80%9A%E1%80%AC%E1%80%B8:%E1%80%A1%E1%80%B1%E1%80%AC%E1%80%B7%E1%80%96%E1%80%BA%E1%80%9C%E1%80%AD%E1%80%AF%E1%80%84%E1%80%BA%E1%80%B8
